djangocon/settings.py

```diff
@@ -158,7 +158,7 @@
 
 
 if DEBUG:
-    # Use `python -m http.server 8888` from the uploads/ directory to serve
+    # Use `python -m SimpleHTTPServer 8888` from the uploads/ directory to serve
```

Comment:
`python -m http.server` is how it's done on Python 3 (which is what the website uses)